### PR TITLE
change the README.md to use examples with oc/podman

### PR DESCRIPTION
**Pull request by pkubatrh**

### CODE REVIEW

**Comment by pkubatrh** [comment]
[test]

**Comment by pkubatrh** [comment]
Added same example as in https://github.com/sclorg/mysql-container/pull/254

**Comment by pkubatrh** [comment]
Looks like an s2i-core image update is needed for Fedora. Opened up PRs downstream.

**Comment by pkubatrh** [comment]
@hhorak @kubco2 PTAL

File: 10.3/root/usr/share/container-scripts/mysql/README.md [hunk 1]
```diff
@@ -2,12 +2,13 @@ MariaDB 10.3 SQL Database Server Docker image
...
 
 This container image includes MariaDB 10.3 SQL database server for OpenShift and general usage.
-Users can choose between RHEL and CentOS based images.
-The RHEL image is available in the [Red Hat Container Catalog](https://access.redhat.com/containers/#/registry.access.redhat.com/rhel8/mariadb-103)
-as registry.access.redhat.com/rhel8/mariadb-103.
-After the CentOS 8 will be released, CentOS image will be available on [Docker Hub](https://hub.docker.com/r/centos/)
-as centos/mariadb-103-centos8.
+Users can choose between RHEL, CentOS and Fedora based images.
+The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
+the CentOS images are available on [Docker Hub](https://hub.docker.com/r/centos/),
```
Comment (kubco2) [inline 10.3/root/usr/share/container-scripts/mysql/README.md]: We do not have centos images. See #98

Comment (pkubatrh) [reply 10.3/root/usr/share/container-scripts/mysql/README.md]: This is just a generic message to point the people looking for a CentOS image to the docker registry. It is true that currently we do not have any 10.3 CentOS based images in the registry but when we decide to ship a RHEL7 based image, there will be a CentOS based image as well. So we might as well have the docs for it already.

**Comment by kubco2** [comment]
otherwise LGTM

**Pull request merged**